vector/src/main/java/im/vector/app/core/platform/VectorBaseActivity.kt

```diff
@@ -260,17 +260,15 @@ abstract class VectorBaseActivity<VB : ViewBinding> : AppCompatActivity(), Maver
 
     private fun handleGlobalError(globalError: GlobalError) {
         when (globalError) {
-            is GlobalError.InvalidToken ->
+            is GlobalError.InvalidToken         ->
                 handleInvalidToken(globalError)
             is GlobalError.ConsentNotGivenError ->
-                consentNotGivenHelper.displayDialog(
-                        globalError.consentUri,
-                        activeSessionHolder.getActiveSession().sessionParams.homeServerHost ?: ""
-                )
-            is GlobalError.CertificateError ->
+                consentNotGivenHelper.displayDialog(globalError.consentUri,
+                        activeSessionHolder.getActiveSession().sessionParams.homeServerHost ?: "")
```

Comment:
Looks like this change in the "revert" commit is not expected. Can you double check that this file is properly formatted?

Reply:
Again caused by long line wrapping. Extracted it out into a private function to make it work with formatting and just generally look cleaner

Reply:
I was not clear: there should be a line break after `(`, and another one before `)`